build/build.sh

```diff
@@ -211,9 +209,12 @@ rocm-*)
         ROCM_DEVICE_LIBS_BRANCH=${VERSION}
         ROCM_DEVICE_LIBS_URL=https://github.com/ROCm/ROCm-Device-Libs.git
     fi
+    # Does not compile with 9.2
+    GCC_VERSION=9.4.0
     URL=https://github.com/ROCm/llvm-project.git
-    LLVM_ENABLE_PROJECTS="clang;lld;clang-tools-extra;compiler-rt"
-    CMAKE_EXTRA_ARGS+=("-DLLVM_TARGETS_TO_BUILD=AMDGPU;X86")
+    LLVM_ENABLE_PROJECTS="clang;lld;clang-tools-extra"
+    LLVM_ENABLE_RUNTIMES+=";libunwind;compiler-rt"
+    CMAKE_EXTRA_ARGS+=("-DLLVM_TARGETS_TO_BUILD=AMDGPU;X86" "-DLLVM_ENABLE_ASSERTIONS=1")
```

Comment:
Do these changes work for all `rocm-` versions including the older ones?

Reply:
yes. I will make changes so that old versions are not affected